db/migrate/20210428203135_add_bio_to_workers.rb

```diff
@@ -0,0 +1,5 @@
+class AddBioToWorkers < ActiveRecord::Migration[6.0]
```

Comment:
Hey @ncbui, I think you also added the workers bio db change and migration to this PR. I just merged your PR that adds the workers_bio field to the database; perhaps update your fork and adjust this PR to (1) delete the duplicate migration file, so it doesn't perform the workers_bio field add a second time? 😅

Reply:
oops, sorry about that. updating PR